src/Export/Adapters/VariantConfigurationAdapter.php

```diff
@@ -29,30 +29,26 @@ public function getOptionAttributes(ProductEntity $product): array
     {
         $options = $product->options;
 
-        $isVariant = !is_null($product->parentId);
-
         if (!$options->count()) {
             return [];
         }
 
-        if ($isVariant) {
-            $variantlisting = array_filter(
-                $product->variantListingConfig['configuratorGroupConfig'] ?? [],
-                function (array $listing) {
-                    return $listing['expressionForListings'];
-                },
-            );
-            $variantListingGroupId = array_map(fn ($listing) => $listing['id'], $variantlisting);
-
-            if (
-                $this->pluginConfig->getMainVariant() === MainVariant::SHOPWARE_DEFAULT &&
-                count($variantlisting) &&
-                !$product->variantListingConfig['displayParent']
-            ) {
-                $options = $options->filter(function (PropertyGroupOptionEntity $option) use ($variantListingGroupId) {
-                    return !in_array($option->groupId, $variantListingGroupId);
-                });
-            }
+        $variantlisting = array_filter(
+            $product->variantListingConfig['configuratorGroupConfig'] ?? [],
+            function (array $listing) {
+                return $listing['expressionForListings'];
+            },
+        );
+        $variantListingGroupId = array_map(fn ($listing) => $listing['id'], $variantlisting);
+
+        if (
+            $this->pluginConfig->getMainVariant() === MainVariant::SHOPWARE_DEFAULT &&
+            count($variantlisting) &&
+            !$product->variantListingConfig['displayParent']
```

Comment:
Aren't we ignoring the `displayParent` configuration for the general export with Shopware Default anyway?
So if we fetch the products while ignoring this config, then we can also do it here for consistency.